src/main/java/bio/terra/app/usermetrics/UserLoggingMetrics.java

```diff
@@ -45,4 +46,9 @@ public void setAll(HashMap<String, Object> value) {
     properties.putAll(value);
     metrics.set(properties);
   }
+
+  @VisibleForTesting
+  public void reset() {
+    metrics.get().clear();
+  }
```

Comment:
This method looks like it's only been added for test initialization.  Can we remove it, and instead call `get().clear()`?